release-notes-v0.5.0.md

```diff
@@ -0,0 +1,67 @@
+  * EPID Verificatio Service: [epid-verification-service](https://github.com/secure-device-onboard/epid-verification-service) is a wrapper service written on top of EPID SDK to assist FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
+  * Test: [test-fidoiot](https://github.com/secure-device-onboard/test-fidoiot) implements a test-suite that gets executed as part of continuous integration pipeline.
+
+### New Features
+
+**pri-fidoiot**: The core protocol implementation supports the following cryptographic algorithms:
+  * Signing keys: ECDSA NIST P-256, ECDSA NIST P-384, RSA2048RESTR, Intel EPID 1.1
+  * Key Exchanges: ECDH256, ECDH384, ASYMKEX2048
+  * Ciphers: AES128/CTR/HMAC-SHA256, AES128/CBC/HMAC-SHA256, AES256/CTR/HMAC-SHA384, AES256/CBC/HMAC-SHA384, AES-CCM-64-128-128, AES-CCM-64-128-256, AES128GCM, AES256GCM, and RSA/NONE/OAEPWithSHA256AndMGF1Padding
+
+**client-sdk**: The core protocol implementation supports the following cryptographic algorithms:
+  * Signing keys: ECDSA NIST P-256, ECDSA NIST P-384
+  * Key Exchanges: ECDH256, ECDH384
+  * Ciphers: AES128/CTR/HMAC-SHA256, AES128/CBC/HMAC-SHA256, AES256/CTR/HMAC-SHA384, AES256/CBC/HMAC-SHA384
+
+**pri-fidoiot**, **client-sdk-fidoiot**: A new ServiceInfo module (fdo_sys) has been implemented. The maximum number of bytes that can be sent in a single ServiceInfo message is configurable.
+
+**pri-fidoiot**, **client-sdk-fidoiot**: Implemented RVBypass feature wherein the TO0 and TO1 protocols are skipped and the device connects to the owner for TO2 protocol using the URLs provided in Rendezvous Directives.
+
+**pri-fidoiot**: Packages have been renamed from org.fido.iot.xxx to org.fidoalliance.fdo.xxx.
+
+**pri-fidoiot**: The owner service can now provide a different owner key that can be used as the manufacturer key after resale.
+
+**pri-fidoiot**: Support for Intel OnDie ECDSA based device attestation has been included.
+
+**epid-verification-service**: It exposes following REST end-points.
+  * Endpoints to retrieve EPID resources such as privrl, sigrl, public key
+  * Endpoints to get EPID signature verification proof result
+  * Endpoints to download cryptomaterial tar ball and cryptomaterial signature files
+
+**epid-verification-service**: Provides Docker Compose based scripts to deploy the services.
+
+**test-fidoiot**: Provides the TestNG based test-suite that is executed for validating the pull requests raised for `pri-fidoiot` and `client-sdk-fidoiot` repositories.
+
+### Changes to existing features
+
+**pri-fidoiot**: Source code has been refactored to separate core protocol implementation and example implementations.
+
+**pri-fidoiot**: TLS support has been enabled for the Docker containers provided within `components-samples\demo`.
+
+### Discontinued features
+
+None
+
+### Fixed Issues
+
+None
+
+### Known Issues
```

Comment:
Do you want to update any known issues from the outstanding defects? If so, we can review the defects and put few incase...